custom_types.yaml

```diff
@@ -587,12 +587,12 @@ node_types:
     interfaces:
       Standard:
         configure:
-          implementation: mesos/mesos_master_install.yml
+          implementation: artifacts/mesos/mesos_master_install.yml
           inputs:
-            consul_server_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_host_list_ips: { get_attribute: [ HOST, public_address ] }
-            zookeeper_peers_ips: { get_attribute: [ HOST, public_address ] }
-            mesos_masters_list_ips: { get_attribute: [ HOST, public_address ] }
+            consul_server_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_host_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            zookeeper_peers_ips: [ { get_attribute: [ HOST, public_address ] } ]
+            mesos_masters_list_ips: [ { get_attribute: [ HOST, public_address ] } ]
```

Comment:
I can do the work btw, I'll just need you to test if it doesn't break anything on your side.